ios/FluentUI/Avatar/AvatarTokens.swift

```diff
@@ -69,13 +69,13 @@ open class AvatarTokens: ControlTokens {
     open var ringDefaultColor: DynamicColor {
         switch style {
         case .default, .group:
-            return globalTokens.brandColors[.tint10]
+            return aliasTokens.colors[.brandStroke1]
```

Comment:
Have you also validated all the changes in the SwiftUI Demo controller?
We don't expect any issues, but just for the sake of validation (which you can also mark in the validation checklist of the PR).

Reply:
Ah I'm sorry I didn't validate the changes on the SwiftUI Demo by the time I made the PR but I just did and everything looks good!